cartridge_account/src/interface.cairo

```diff
@@ -50,4 +50,18 @@ trait AccountCamelABI<TState> {
     fn getPublicKey(self: @TState) -> felt252;
     fn isValidSignature(self: @TState, hash: felt252, signature: Array<felt252>) -> felt252;
     fn supportsInterface(self: @TState, interfaceId: felt252) -> bool;
+    fn verifyWebauthnSigner(
+        ref self: TState, 
+        pub_x: u256,
+        pub_y: u256, // public key as point on elliptic curve
+        // r: u256, // 'r' part from ecdsa
```

Comment:
Should'n that be uncommented?

Reply:
Yes, probably. As a side note, what is the point of the interface if everything works even though the interface doesn't match the implementation?

Reply:
I've uncommented this.